### PR TITLE
Bunch of updates to the app store documentation

### PR DESCRIPTION
**Pull request by wiswedel**
Signed-off-by: Sascha Wiswedel <sascha.wiswedel@nextcloud.com>

### CODE REVIEW

File: admin_manual/apps_management.rst [hunk 1]
```diff
@@ -21,19 +21,19 @@ During the Nextcloud server installation, some apps are enabled by default.
...
-Apps with **Approved** tag are community-developed and were uploaded in the
-`Nextcloud App Store <https://apps.nextcloud.com>`_.
-
 .. note:: If you would like to develop your own Nextcloud app, you can find out
-          more information in our `developer manual <https://docs.nextcloud.com/server/14/go.php?to=developer-manual>`_.
+          more information in our `developer manual <https://docs.nextcloud.com/server/latest/go.php?to=developer-manual>`_.
 .. TODO ON RELEASE: Update version number above on release
 
-All apps need to be licensed under AGPLv3+ or any compatible license.
+All apps must be licensed under AGPLv3+ or any compatible license.
```
Comment (kesselb) [inline admin_manual/apps_management.rst:36]: Mind to add that information also to https://github.com/nextcloud/documentation/blob/master/developer_manual/app/publishing.rst?

Comment (wiswedel) [reply admin_manual/apps_management.rst:36]: > Mind to add that information also to https://github.com/nextcloud/documentation/blob/master/developer_manual/app/publishing.rst?

sure, here you go: https://github.com/nextcloud/documentation/pull/1969/commits/5b06803335bdd47be2fe4d4e5a791b4ca9af09c2

**Pull request merged**